libraries/botbuilder/src/teamsInfo.ts

```diff
@@ -31,6 +35,36 @@ export class TeamsInfo {
         return await this.getTeamsConnectorClient(context).teams.fetchTeamDetails(t);
     }
 
+    public static async sendMessageToTeamsChannel(context: TurnContext, activity: Partial<Activity>, teamsChannelId: string): Promise<[Partial<ConversationReference>, string]> {
+        if (context == null){

```

Comment:
This does not follow our typical JavaScript conventions. You should test for Falsey. i.e. you can use if (!context) etc.

Reply:
...and all these if statements are missing a space between the closing parenthesis and the opening curly brace.